doc/02-installation.md.d/For-Container.md

```diff
@@ -0,0 +1,179 @@
+to do this, you can either mount the `/etc/msmtprc` file from your host system into the container or provide the
+necessary configuration for the `~/icinga/.msmtprc` file via the `MSMTPRC` environment variable.
+
+## Environment Variables
+
+Most of the environment variables are used as parameters for the `icinga2 node setup` command. If you set any of these
+variables, and the `node setup` has not been run yet, the entrypoint script will automatically run the command for you.
+
+The following environment variables are available:
+
+| Variable                                                 | Node setup CLI                                      |
+|----------------------------------------------------------|-----------------------------------------------------|
+| `ICINGA_ACCEPT_COMMANDS=1`                               | `--accept-commands`                                 |
+| `ICINGA_ACCEPT_CONFIG=1`                                 | `--accept-config`                                   |
+| `ICINGA_DISABLE_CONFD=1`                                 | `--disable-confd`                                   |
+| `ICINGA_MASTER=1`                                        | `--master`                                          |
+| `ICINGA_CN=icinga-master`                                | `--cn icinga-master`                                |
+| `ICINGA_ENDPOINT=icinga-master,2001:db8::192.0.2.9,5665` | `--endpoint icinga-master,2001:db8::192.0.2.9,5665` |
+| `ICINGA_GLOBAL_ZONES=global-config`                      | `--global_zones global-config`                      |
+| `ICINGA_LISTEN=::,5665`                                  | `--listen ::,5665`                                  |
+| `ICINGA_PARENT_HOST=2001:db8::192.0.2.9,5665`            | `--parent_host 2001:db8::192.0.2.9,5665`            |
+| `ICINGA_PARENT_ZONE=master`                              | `--parent_zone master`                              |
+| `ICINGA_TICKET=0123456789abcdef0123456789abcdef01234567` | `--ticket 0123456789abcdef0123456789abcdef01234567` |
+| `ICINGA_ZONE=master`                                     | `--zone master`                                     |
+
+Special variables:
+
+* `ICINGA_TRUSTEDCERT`'s value is written to a temporary file in the container, which is then used
+  as the `--trustedcert` parameter for the `icinga2 node setup` command.
+* `ICINGA_CACERT`'s value is directly written to the `/var/lib/icinga2/certs/ca.crt` file in the container.
+
+## Build Your Own Image
+
+If you want to build your own Icinga 2 container image, you can use the provided `Containerfile` in the Icinga 2
+repository. Additionally, there's a `docker-build.sh` helper script that simplifies the build process. The parameters
+used below for the script are explained within the script itself, so you can adjust them as needed.
+
+```
+git clone https:github.com/Icinga/icinga2.git
+cd icinga2
+./tools/container/docker-build.sh load
```

Comment:
`load` is already the default action:

```suggestion
./tools/container/docker-build.sh
```

Though it should also be mentioned that this will build an image called `icinga/icinga2:test` somewhere. Probably it would make sense to have a command here that shows how to change that name.